packages/xo-server/src/xapi-object-to-xo.mjs

```diff
@@ -193,6 +197,9 @@ const TRANSFORMS = {
       logging: obj.logging,
       name_description: obj.name_description,
       name_label: obj.name_label,
+      maintained:
+        MAINTAINED_HOST_VERSION[softwareVersion.product_brand]?.some(v => v === softwareVersion.product_version) ??
```

Comment:
If the host version is higher than 8.2, is it still not maintained?

You can use `semver.satisfies`.

Reply:
Is that not better to just add the new available version in the `maintainedVersion.json` ?